src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/msg/EnvelopeMsg.java

```diff
@@ -139,6 +146,17 @@ public String toString() {
             + "}";
     }
 
+    private List<String> retrieveOcrValidationWarnings(Envelope envelope) {
```

Comment:
maybe this and `retrieveOcrData` below could use the same function to the the relevant scannable item

Reply:
done